src/client/ui/admin.js

```diff
@@ -66,12 +65,7 @@ export function getIframe(data) {
 }
 
 export function getActionLogger(data) {
-  const { actions = [] } = data;
-  const log = actions
-    .map(action => stringify(action, null, 2))
-    .join('\n\n');
-
-  return (<ActionLogger actionLog={log} onClear={clearLogs} />);
+  return (<ActionLogger data={data} onClear={clearLogs} />);
```

Comment:
Here do not send the whole data, but just send an array of strings. We can do it like this:

```js
export function getActionLogger(data) {
  const { actions = [] } = data;
  const logs = actions
    .map(action => stringify(action, null, 2));

  return (<ActionLogger logs={logs} onClear={clearLogs} />);
}
```